packages/app/src/generated/graphql.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable */
+import type { TypedDocumentNode as DocumentNode } from '@graphql-typed-document-node/core';
+export type Maybe<T> = T | null;
+export type Exact<T extends { [key: string]: unknown }> = { [K in keyof T]: T[K] };
+export type MakeOptional<T, K extends keyof T> = Omit<T, K> & { [SubKey in K]?: Maybe<T[SubKey]> };
+export type MakeMaybe<T, K extends keyof T> = Omit<T, K> & { [SubKey in K]: Maybe<T[SubKey]> };
+/** All built-in and custom scalars, mapped to their actual values */
+export type Scalars = {
+  ID: string;
+  String: string;
+  Boolean: boolean;
+  Int: number;
+  Float: number;
+  /** A date-time string at UTC, such as 2007-12-03T10:15:30Z, compliant with the `date-time` format outlined in section 5.6 of the RFC 3339 profile of the ISO 8601 standard for representation of dates and times using the Gregorian calendar. */
+  DateTime: any;
+  /** The `JSON` scalar type represents JSON values as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf). */
+  JSON: any;
+};
+
+
+
+export type BrowserFamily =
+  | 'chromium'
+  | 'firefox';
+
+
+
+export type FrontendFramework =
+  | 'cra'
+  | 'nextjs'
+  | 'nuxtjs'
+  | 'react'
+  | 'vue'
+  | 'vuecli';
+
+
+
+
+export type NavItem =
```

Comment:
I don't know why these came along for the ride when I generated the types for  `packages/app`? These are only used in `launchpad`. 🤷 not a big deal but would be nice to know why they got included.

Reply:
Pretty sure the enums are always added with the `typescript` definition even if they're not used. Can't remember if there's a way to customize this, I think there is but will need to check.